### PR TITLE
Enhancing README file with notes

### PR DESCRIPTION
**Pull request by davidverdu**
Detailing some steps to run the example better

## Description
enhancing the documentation

## Changes
README file

## Checks
- [x] Project Builds
- [x] Project passes tests and checks
- [x] Updated documentation accordingly

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -480,6 +480,12 @@ public handle(register: Register): void {
 }
 ```
 
+Remember to import the event class correctly on the top part of the file:
```
Comment (javiertoledo) [inline docs/README.md:483]:
```suggestion
Remember to import the event class correctly on the top of the file:
```

File: docs/README.md [hunk 2]
```diff
@@ -613,7 +619,12 @@ Booster applications provide you with a GraphQL API out of the box. You send com
 _mutations_ and get read models data using _queries_ or _subscriptions_. 
 
 In this section, we will be sending requests by hand using the online tool [Postwoman](https://postwoman.io/graphql), 
-which is free and includes great support for GraphQL. However, you can use any HTTP client you want.
+which is free and includes great support for GraphQL. However, you can use any HTTP client you want. In Postwoman
+online tool, you should enter the following URL as Endpoint:
```
Comment (javiertoledo) [inline docs/README.md:623]:
```suggestion
which is free and includes great support for GraphQL. However, you can use any client you want. Your endpoint URL should look like this:
```

File: docs/README.md [hunk 3]
```diff
@@ -613,7 +619,12 @@ Booster applications provide you with a GraphQL API out of the box. You send com
...
+
+```text
+https://<some random number>.execute-api.us-east-1.amazonaws.com/production/graphql
```
Comment (alvaroloes) [inline docs/README.md:626]: This is specific to AWS. I think we should not use anything that is provider-specific if we can avoid it. 
We already talked about the `httpURL` output, so we can rewrite this like:
```suggestion
<httpURL>/graphql
```

**Comment by davidverdu** [comment]
I agree with you. Your suggestions make sense.

File: docs/README.md (hunk 1 above)
Comment (davidverdu) [reply docs/README.md]: This make sense Javi

**Pull request merged**